src/txpool/getters.yaml

```diff
@@ -0,0 +1,69 @@
+- name: txpool_transactions
+  summary: Returns a list of all transactions that match the supplied filter conditions that are either pending for inclusion in the next block(s) or scheduled for future execution.
+  params:
+    - name: Filter
+      schema:
+        title: txpool filter
+        type: object
+        properties:
+          from:
+            title: from address
+            $ref: '#/components/schemas/address'
+          to:
+            title: to address
+            $ref: '#/components/schemas/address'
+          gas:
+            title: gas limit
+            $ref: '#/components/schemas/FilterOperators'
+          gasPrice:
+            title: gas price
+            $ref: '#/components/schemas/FilterOperators'
+          maxFeePerGas:
+            title: max fee
+            $ref: '#/components/schemas/FilterOperators'
+          maxPriorityFeePerGas:
+            title: max priority fee
+            $ref: '#/components/schemas/FilterOperators'
+          value:
+            title: value
+            $ref: '#/components/schemas/FilterOperators'
+          nonce:
+            title: nonce
+            $ref: '#/components/schemas/FilterOperators'
+  result:
+    name: Result
+    schema:
+      title: Result
+      schema:
+        $ref: '#/components/schemas/TxPoolTransactions'
+        title: transactions
+        type: object
+        properties:
+          pending:
+            title: pending
+            type: array
+            items:
+              $ref: '#/components/schemas/GenericTransaction'
```

Comment:
Just to start a discussion, since specially with EIP-4844, we could have quite big network representation (that includes blob content) of a tx, so a way to select what to return and/or to limit the amount of data returned should be considered.

Reply:
From my understanding, 4844/blobs will have its own transaction pool and do not use the generic tx pool. So not sure if this should be part of this proposal.

Reply:
Quoting the previous answer, where I think blob txs should be included, setting some kind of limits make sense